### ISSUE TITLE
LogicalId, Version... are mandatory parameters for Set-IshPublicationOutput when trying to do a batch update using "|" symbol

### ISSUE THREAD

**Issue by janb87** [sdl/ISHRemote#18]
I was trying to update metadata for a publication output. I'm facing an issue that I cannot use the option to pipe the data of the get commandlet to the set commandlet. This is because some parameters are mandatory for the commandlet.

```powershell
$pubOutputs =  $pubs = Get-IshPublicationOutput -IshSession $session -LogicalId "GUID-..."

if ($pubOutputs.count -gt 0) {
   $pubOutputs | Set-IshPublicationOutput -IshSession $session -Metadata $updatedMetadata
}
```

@ddemeyer If you are ok with it I can provide a pull request to fix it.

**Comment by ddemeyer** [comment]
Please go ahead Jan ☺

**Comment by ddemeyer** [comment]
So `IshObjectGroup` parameterset expects correct `IshObject`s to come in. The request here is to have `Set-IshPublicationOutput` do the metadata merge on every incoming `IshObject`.
Workaround could be 
```
$publicationOutputIshObjects | 
Set-IshMetadataField -IshSession $ishSession -Name FCHANGES -Level Version -Value "Converted" | 
Set-IshPublicationOutput -IshSession $ishSession 
```

**Comment by ddemeyer** [comment]
Closed by #40
